arbitrum-docs/node-running/reference/software-releases.md

```diff
@@ -0,0 +1,58 @@
+---
+title: 'ArbOS software releases'
+sidebar_label: 'ArbOS software releases'
+author: dlee
+---
+
+:::info
+This page is meant to give a high level summary of node software releases, with particular emphasis on releases that bump the ArbOS version. To find information on all releases of Nitro, including the latest release, and accompanying changelogs, please visit the [Nitro Github Repository](https://github.com/OffchainLabs/nitro/releases).
+
+:::
+
+The name of the node software that is used to power Arbitrum is called Nitro. While Nitro releases can get shipped as frequently as multiple times per week, there are a few Nitro releases that carry an upgrade to [ArbOS](https://forum.arbitrum.foundation/t/arbitrum-arbos-upgrades/19695) - the Layer 2 EVM hypervisor that facilitates the execution environment of an Arbitrum chain. Releases of ArbOS can be considered as Arbitrum's equivalent to a hard fork and are special because they could alter a node's ability to produce valid Arbitrum blocks. Due to the impact that an ArbOS upgrade has on an Arbitrum chain, ArbOS upgrades are **mandatory to pick up** as a validator of a public Arbitrum chain. Note that every Nitro release is backwards compatible (i.e. supports all previous versions of ArbOS), which means your node will need to be at a Nitro version equal to or greater than the Nitro version that supports a given ArbOS release.
+
+ArbOS upgrades are carried out by the chain's owner; in the case of Arbitrum One and Nova, the owner is the Arbitrum DAO and so an upgrade will require a governance proposal and vote to pass to complete the upgrade. [This is an example of a Nitro release that contains an ArbOS version bump, specifically to ArbOS 11](https://github.com/OffchainLabs/nitro/releases/tag/v2.2.0).
+
+ArbOS is an essential part of the Nitro stack. Visit [this page to learn more about Nitro's architecture](../../inside-arbitrum-nitro/inside-arbitrum-nitro.mdx) and go [here to learn more about ArbOS](https://forum.arbitrum.foundation/t/arbitrum-arbos-upgrades/19695).
```

Comment:
```suggestion
Visit [Inside Arbitrum Nitro](../../inside-arbitrum-nitro/inside-arbitrum-nitro.mdx) to learn more about Nitro's architecture; more information about ArbOS software releases is available on [the Arbitrum DAO forum](https://forum.arbitrum.foundation/t/arbitrum-arbos-upgrades/19695).
```

Nit - whenever possible, describe the link so readers know where they're headed, without having to hover over the link.